### PR TITLE
Ceres 476 add custom user

### PR DESCRIPTION
**Pull request by abhilashjoseph**

### CODE REVIEW

**Comment by abhilashjoseph** [comment]
> Nice one. Could you squash 5 latest commits before you merge please?
Will squash for sure!!

File: cmd/azure-keyvault-secrets-webhook/main.go [hunk 1]
```diff
@@ -88,11 +88,11 @@ func getInitContainers() []corev1.Container {
 	cmd := "cp /usr/local/bin/azure-keyvault-env /azure-keyvault/"
 
 	if !config.customAuth {
-		cmd = cmd + fmt.Sprintf(" && cp %s %s ", config.cloudConfigHostPath, config.cloudConfigContainerPath)
+		cmd = cmd + fmt.Sprintf(" && cp %s %s && ", config.cloudConfigHostPath, config.cloudConfigContainerPath)
+		cmd = cmd + fmt.Sprintf("chmod -R a+w %s && ", "/azure-keyvault")
+		cmd = cmd + fmt.Sprintf("chmod 444 %s", config.cloudConfigContainerPath)
```
Comment (bambriy) [inline cmd/azure-keyvault-secrets-webhook/main.go]: 444 is back?

Comment (abhilashjoseph) [reply cmd/azure-keyvault-secrets-webhook/main.go]: testing it now, i believe the a+w didn't give enough permissions for the exe to read the configs. 
level=fatal msg="env-injector: failed to get credentials for azure key vault, error failed to read cloud config file in an effort to get credentials for azure key vault, error: open /azure-keyvault/azure.json: permission denied"

Comment (abhilashjoseph) [reply cmd/azure-keyvault-secrets-webhook/main.go]: or may be have a+rw

Comment (bambriy) [reply cmd/azure-keyvault-secrets-webhook/main.go]: makes sense

**Pull request merged**